es/lib/components/modal.js

```diff
@@ -80,6 +80,8 @@ function registerPostMessageCallback() {
       remove();
     } else if (event.action == 'complete' && event.type == 'modal') {
       redirect(event.options);
+    } else if (event.action == 'creditkey-event' && event.type == 'modal') {
+      console.log('Credit Key Event:', event.options);
```

Comment:
does this need a re-build to get rid of this? same thing in`lib/lib/components/modal.js`